### DISCUSSION TITLE
Is this computer compatible with LivelyWallpaper?

### DISCUSSION THREAD

**Discussion by JJHilvick21** [rocksdanister/lively#929] [Q&A]
HP Laptop 15z-gw000:

specs:
Windows 11 Home
AMD Athlon™ Gold 3150U (2.4 GHz base clock(2h), up to 3.3 GHz max boost clock(2i), 4 MB L3 cache, 2 cores) + AMD Radeon™ Graphics
8 GB DDR4-2400 SDRAM (1 x 8 GB)
15.6" diagonal HD, SVA, BrightView, micro-edge, WLED-backlit, 220 nits (1366 x 768)
1 TB 5400 rpm SATA

**Comment by rocksdanister** [comment]
Yes, I recommend sticking with simple 1080p/720p video wallpapers:
https://github.com/rocksdanister/lively/wiki/Performance
